frontend/src/Components/Visulaizations/BarGraph.js

```diff
@@ -0,0 +1,82 @@
+import { Paper } from '@mui/material';
+import { Bar } from 'react-chartjs-2';
+// Chart.register(Bar);
+
+
+export default function BarVis(){
+    
+    const labels = ["Jan", "Feb", "mar", "apr", "may", "jun", "jul"];
```

Comment:
Tricky question, how are you going to pass this data long?

Reply:
I will pass the data I get from backend as props to these components.